vertical-pod-autoscaler/pkg/updater/main.go

```diff
@@ -66,18 +62,10 @@ var (
 
 	evictionRateBurst = flag.Int("eviction-rate-burst", 1, `Burst of pods that can be evicted.`)
 
-	address         = flag.String("address", ":8943", "The address to expose Prometheus metrics.")
```

Comment:
Address is removed from each component and added to the common flags, but each component has a different default. 
I think this behaviour shouldn't change.

Reply:
Will be fixed too.